test/test_kubeclient.rb

```diff
@@ -106,11 +106,11 @@ def test_get_all
     assert_instance_of(EntityList, result['pod'])
```

Comment:
Yes, I didn't change it yet because I sent the other PR.  But now that it's merged, I can fix this.